frappe/public/js/frappe/ui/toolbar/search.js

```diff
@@ -372,6 +400,22 @@ frappe.search.SearchDialog = Class.extend({
 					});
 			}
 		},
+		global_tag: {
+			input_placeholder: __("Search"),
+			empty_state_text: __("Search for anything"),
+			no_results_status: (keyword) => __("<p>No results found for '" + keyword + "' in Global Tags</p>"),
```

Comment:
This translation string wont work. Also, remove html tags from it.

https://frappe.io/docs/user/en/translations